### PR TITLE
feat: [BREAKING CHANGE] upgrade ali-oss to support async function

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

Closes https://github.com/eggjs/egg/issues/3071

### CODE REVIEW

**Comment by popomore** [comment]
再看看

File: .travis.yml [hunk 1]
```diff
@@ -25,3 +25,6 @@ env:
...
     # ALI_SDK_STS_BUCKET
     - secure: "EVVuk1PbyDXRUdn6BySsmb50FTzEVpFyUlhMpKP/WOXIXdET2ssqN6mUPw2KXZZhCMAFNrdXEHI5Cey8SLgECIvzSB+YVAED7k6yD6IrkWNh4gtKA5EF77ZdBIPAFdeJnvXbZk6N4Pusx+cXvSDtLfBzwgRrASgecz7KXq9wnZjkmGC//+7SwfXK9KefZvLIh2wJWFqQXSazXCgGH4GAiwrLonX5le8K+XbdQCE2FoCFRVgvwl+2w7go6qlpdOXuFAvVqhmaNw2MK7258ZEudQLKD/oXvkO8Lqxizr4DvjZnrxejKpL36KJoDp/eXZt2litiDUSxnS0wREdtP0MO5zww6SrAKR8t8JN/WsWPvBBuELe8+I98ZAZag5e5b5XLCUWh+PpJOqpYMN5KPI78GyTtUMXUHAICljagCRtPhVP6xqTlWzC9mPWcQIEi/K04b9bxxk2vHZzOLEuvnMXg9NYN/t7XKtKNk1hNruboJ3yYDjq9XawpviY8SmvR9xjwVThLNIrpLfVNglDuS5ZKwdhEEBDX4qAFfMng6EBxTaljafUjw8UBZTEIaqSsV2e40MFFNUD8s+Jz6Rnt1CHqkkF8374dFLBWwljW72Pntqfw9jydzt93nT6SucY/o16zdhPjTdf5b90kXZLZrhBdTevdLmQhA5rdr3bl/+tLnBI="
+
```
Comment (dead-horse) [inline .travis.yml]: 怎么多了三个空行..

**Comment by popomore** [comment]
改了

**Comment by popomore** [comment]
+ egg-oss@2.0.0

**Pull request merged**